keps/sig-apps/3329-retriable-and-non-retriable-failures/README.md

```diff
@@ -1815,7 +2109,10 @@ previous answers based on experience in the field.
 
 ###### Will enabling / using this feature result in any new API calls?
 
-Yes. An API call to append a Pod condition when deleting the Pod.
+Yes. An PATCH API call to append a Pod condition when deleting the Pod. Also,
```

Comment:
nit: s/An PATCH/A PATCH/

Should not hold up the PR.

Reply:
Done